docs/framework/winforms/windows-forms-overview.md

```diff
@@ -61,7 +61,7 @@ For step-by-step information about how to use these features, see the following
 |-----------------|----------------|
 |Using the <xref:System.Windows.Forms.BindingSource> component|[How to: Bind Windows Forms Controls with the BindingSource Component Using the Designer](../../../docs/framework/winforms/controls/bind-wf-controls-with-the-bindingsource.md)|
 |Working with [!INCLUDE[vstecado](../../../includes/vstecado-md.md)] data sources|[How to: Sort and Filter ADO.NET Data with the Windows Forms BindingSource Component](../../../docs/framework/winforms/controls/sort-and-filter-ado-net-data-with-wf-bindingsource-component.md)|
-|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://msdn.microsoft.com/library/f6e08c2c-c792-43de-adf3-3e52c0100225)|
+|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/wwh8ka92(v=vs.120))|
```

Comment:
Let's change this to go to https://docs.microsoft.com/visualstudio/data-tools/bind-windows-forms-controls-to-data-in-visual-studio instead